org.gnome.Crosswords.PuzzleSets.oedipus.json

```
@@ -0,0 +1,34 @@
+{
+	"app-id": "org.gnome.Crosswords.PuzzleSets.oedipus",
+	"runtime": "org.gnome.Crosswords",
+	"runtime-version": "stable",
+	"branch": "1",
+	"sdk": "org.gnome.Sdk//42",
```

```suggestion
	"sdk": "org.gnome.Sdk//43",
```

the app used the 43 SDK